assets/src/edit-story/components/checklist/checklistContent/priorityChecks.js

```diff
@@ -56,7 +68,8 @@ export function PriorityChecks({ isOpen, onClick, title }) {
         <StoryTitleLength />
         <StoryMissingExcerpt />
         <StoryPosterAttached />
-        <StoryPosterPortraitSize />
+        {/* TODO: this overlaps alot with aspect ratio, do we need both? */}
+        {/* <StoryPosterPortraitSize /> */}
```

Comment:
@embarks eyes here would be great, I feel like something is wrong but can't pinpoint it.

Reply:
poster size:
```
return (
    story.featuredMedia?.height < FEATURED_MEDIA_RESOURCE_MIN_HEIGHT ||
    story.featuredMedia?.width < FEATURED_MEDIA_RESOURCE_MIN_WIDTH
  );
```

aspect ratio:
```
  const hasCorrectAspectRatio =
    Math.abs(
      story.featuredMedia.width / story.featuredMedia.height -
        ASPECT_RATIO_LEFT / ASPECT_RATIO_RIGHT
    ) <= 0.001;

  return !hasCorrectAspectRatio;
```

they are pretty different checks but they rely on the same properties and probably have the same highlight behavior. i think UX wise it would be better to combine AspectRatio and PortraitSize but it might make the copy logic kinda weird is all.

Reply:
Right on. I went ahead and uncommented this out and added a todo comment with this new issue: #8129 to look into it later. Since these have always existed in tandem there's no harm in leaving it for the time being.